iothub/service/src/Configurations/ConfigurationContent.cs

```diff
@@ -12,16 +12,28 @@ namespace Microsoft.Azure.Devices
     public class ConfigurationContent
     {
         /// <summary>
-        /// Gets or sets Module Configurations
+        /// Gets or sets the configurations to be applied to the Edge agent.
         /// </summary>
+        /// <remarks>
+        /// For more information on this field, see <see href="https://docs.microsoft.com/en-us/azure/iot-edge/module-composition?view=iotedge-2020-11#create-a-deployment-manifest">this documentation</see>.
+        /// </remarks>
         [JsonProperty(PropertyName = "modulesContent")]
 #pragma warning disable CA2227 // Collection properties should be read only
         public IDictionary<string, IDictionary<string, object>> ModulesContent { get; set; }
 
 #pragma warning restore CA2227 // Collection properties should be read only
 
         /// <summary>
-        /// Gets or sets Module Configurations
+        /// Gets or sets the configurations to be applied on device modules
+        /// </summary>
+        [JsonProperty(PropertyName = "moduleContent")]
+#pragma warning disable CA2227 // Collection properties should be read only
+        public IDictionary<string, object> ModuleContent { get; set; }
```

Comment:
Interesting, so we never even had it? do we know if this was added at some point and we just never got the message or we originally missed it? Not that it really matters here, I am just curious.

Reply:
I'm not sure. I imagine the SDK team wasn't given a full spec for this feature, and this field was lost in the shuffle somehow